pkgs/tools/misc/coreboot-configurator/default.nix

```diff
@@ -3,34 +3,43 @@
 , fetchFromGitHub
 , inkscape
 , meson
+, mkDerivation
 , ninja
 , pkg-config
+, pkexecPath ? "/run/wrappers/bin/pkexec"
 , yaml-cpp
 , nvramtool
+, systemd
 , qtbase
 , qtsvg
 , wrapQtAppsHook
 }:
 
-stdenv.mkDerivation {
+mkDerivation {
   pname = "coreboot-configurator";
-  version = "unstable-2022-08-22";
+  version = "unstable-2023-01-17";
 
   src = fetchFromGitHub {
     owner = "StarLabsLtd";
     repo = "coreboot-configurator";
-    rev = "37c93e7e101a20f85be309904177b9404875cfd8";
-    sha256 = "2pk+uJk1EnVNO2vO1zF9Q6TLpij69iRdr5DFiNcZlM0=";
+    rev = "944b575dc873c78627c352f9c1a1493981431a58";
+    sha256 = "sha256-ReWQNzeoyTF66hVnevf6Kkrnt0/PqRHd3oyyPYtx+0M=";
   };
 
   nativeBuildInputs = [ inkscape meson ninja pkg-config wrapQtAppsHook ];
   buildInputs = [ yaml-cpp qtbase qtsvg ];
 
   postPatch = ''
     substituteInPlace src/application/*.cpp \
-      --replace '/usr/bin/pkexec' 'sudo' \
-      --replace '/usr/bin/systemctl' 'systemctl' \
+      --replace '/usr/bin/pkexec' '${pkexecPath}' \
```

Comment:
Would it be possible to load this from `$PATH`? That would avoid needing to pass in the absolute path as an argument.

Reply:
@danth good point. I tested this and it seems to work indeed, so I put the path by default now but I left the possibility to override it if needed.